nasa-gibs/worldview
# Fix guitar pick offset in alternate timezones

## Description

Fixes #1410, Fixes #1205

Guitar pick will not move around when in other timezones. This PR should also fix weird "jumping" behavior seen in other timezones and likely the behavior outlined in #1205. 

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc

## Further comments

There PR includes bug fixes for enabling the minute zoom level when using the `showSubdaily` param. The subdaily zoom level selector and inputs were not showing when the param was added.

**To Test**
- Test moving the pick, clicking the boundary, etc. in each zoom level. 
- Change the timezone in "Date & Time" settings on your machine and repeat.

There should be no noticable difference when changing the timezone. 
The guitar pick should be at the start of each tick mark.

@nasa-gibs/worldview
